.gitignore

```diff
@@ -2,6 +2,10 @@
 /docker/mysql
 ### docker ###
 
+###> idea ###
+/.idea
+### idea ###
+
```

Comment:
You should put this lines in a personnal gitignore file.  
[https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file](https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file)